pages/nft/purchase/_iscnId.vue

```diff
@@ -130,15 +137,31 @@ export default class NFTTestButtonPage extends Vue {
       return `(${price.toFixed(3)} USD)`;
   }
 
+  get owningCount() {
+    if (!this.address) return -1
+    return this.history.reduce((count, { toWallet }) => toWallet === this.address ? count + 1 : count, 0)
+  }
+
   async mounted() {
     await Promise.all([
       this.getPurchaseInfo(),
       this.getNftMetadata(),
+      this.getNFTHistory(),
     ]).catch((err) => this.$nuxt.error({ statusCode: 404, message: err }))
     await this.getOwnerName().catch(err => console.error(err))
     this.getLIKEPrice().catch(err => console.error(err))
   }
 
+  async getNFTHistory() {
+    const { data } = await this.$axios.get(API_LIKER_NFT_HISTORY, {
```

Comment:
Can we use chain API instead? less issue more robust (not sure if faster)
https://github.com/likecoin/liker-land/blob/feature/nft/src/pages/nft/class/_classId.vue#L449